webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableSchema.js

```diff
@@ -0,0 +1,100 @@
+import {
+  sortableHeadFormat,
+  nameFormat,
+  actionsFormat,
+  hostsCountFormat,
+} from './ModelsTableFormats';
+import {
+  headerFormatWithProps,
+  headerFormat,
+  cellFormat,
+  cellFormatWithProps,
+  ellipsisFormat,
+} from '../common/table';
+
+export const schema = ({
```

Comment:
IMO it should be named `cretaeModelsTableSchema`.
Also the filename.

Reply:
TBH, I hate this function. I don't think it's readable and it makes me sad to think that I am building an abstraction over something that doesn't need it right now.

I am going to pull it back to the `<ModelsTable>` component. The reason for that is because the schema should be coupled to ModelsTable. However, the `<Table>` component gives a good abstraction which is good enough.